tests/functional/spec/components/radio.spec.js

```diff
@@ -0,0 +1,124 @@
+const helpers = require('../../helpers');
+const RadioMandatoryPage = require('../../pages/surveys/radio/radio-mandatory.page');
+const RadioNonMandatoryPage = require('../../pages/surveys/radio/radio-non-mandatory.page');
+const SummaryPage = require('../../pages/surveys/radio/summary.page');
+
+describe('Component: Radio', function() {
+
+var radio_schema = 'test_radio.json';
+
+
+    describe('Optional', function() {
+          beforeEach('load the survey', function() {
```

Comment:
I thought we were going to keep this as `before` then add an `afterEach` to navigate back a page.
That way it only launches the survey once rather then once per test. which would save lots of time

Reply:
I thought that to but after reading some of the scenarios I would have to navigate back to the start for some of them anyway so just stuck with launching survey from the start for each test.